parm/metplus_config/plots/aqm/grid2obs/py_plotting_ozmax8_headline.config

```diff
@@ -116,27 +117,29 @@ export EVAL_PERIOD="TEST"
 
 # If EVAL_PERIOD="TEST", will use statistics from the valid or init range below. If not, ignores 
 # these settings. DATE_TYPE decides whether valid or init is used.
-export VALID_BEG="$PDYm31"
-export VALID_END="$VDATE"
-export INIT_BEG="$PDYm31"
-export INIT_END="$VDATE"
+
+export VALID_BEG="${PLOT_START}"
+export VALID_END="${PLOT_END}"
+
+export INIT_BEG="${PLOT_START}"
+export INIT_END="${PLOT_END}"
```

Comment:
Why are we no longer using $PDYm31, etc. It seems to me that some of these changes are beyond the scope of an AQMv7 upgrade. @Ho-ChunHuang-NOAA @ShelleyMelchior-NOAA @PerryShafran-NOAA

Reply:
The default remains to be PDYm31 as PLOT_START defined in JEVS_AQM_PLOTS.  I think all values should be defined in the J-jobs but not hard-coded in scripts several layers down from J-jobs.  Also allow user an option to change time period in their own driver scripts for their own testing.

The original code setting is fine in operation (but all the file checking are unnecessary in operational environment by default).  But it runs in risk as if user set VDATE earlier than PDYm31, as I did before to reproduce graphic for presentation, the job will crash.